src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  if(days<3){
+    return days*RENT_COST;
+  } else if(days < 7){
+    return days*RENT_COST-20;
```

Comment:
Prefer using `if` with `return` over `if else` to simplify the code logic. Since the first `if` block contains a `return` statement, the `else` keyword is not necessary here.